src/main/fc/init.c

```diff
@@ -285,13 +285,25 @@ static void configureSPIAndQuadSPI(void)
 }
 
 #ifdef USE_SDCARD
-void sdCardAndFSInit()
+static void sdCardAndFSInit()
 {
     sdcard_init(sdcardConfig());
     afatfs_init();
 }
 #endif
 
+static void swdPinsInit(void)
+{
+    IO_t io = IOGetByTag(IO_TAG(PA13)); // SWDIO
```

Comment:
`DEFIO_TAG_E` instead of `IO_TAG` for targets that doesn't even define these pins as available in `TARGET_IO_PORTA`?

Reply:
Good point, fixed.